packaging/suse/portus.spec.in

```diff
@@ -49,20 +59,37 @@ Obsoletes:      Portus < %{version}
 # This is the reason why we are obsoleting it
 Obsoletes:      Portus = 20151120162040
 
-# Javascript engine to build assets
+# Javascript engine to build assets. Note that yarn-packaging will automatically
+# create the provides for the JS libs
```

Comment:
Doesn't this conflict with the script that is generating all those `Provides ...` ?